### DISCUSSION TITLE
Label drift issue

### DISCUSSION THREAD

**Discussion by bibasaktanesi** [google/model-viewer#3794] [Q&A]
Hello,
My model has one or more tags. GLB works fine, but when I open it here, the label slips. (replaces)

How can I fix?

**Comment by elalish** [comment]
I'm afraid I have no idea what you mean without a picture or repro link.

**Comment by bibasaktanesi** [comment]
In my design, there are no white reflections on the label and the position of the label in the 2nd picture should be on the edge. But when I open it here, the label of the design slips or light reflections occur.

**Comment by bibasaktanesi** [comment]
This situation occurs more when turning

**Reply by elalish** [reply]
I'm still not sure what you mean by "slips" - do you have a good/bad image pair of your label? As for reflections moving as the object turns, that's the whole point of PBR - to make it realistic by dynamically changing based on the lighting. Is there something that seems incorrect?

**Comment by bibasaktanesi** [comment]


**Reply by elalish** [reply]
That's exactly what I was asking for, thanks! I see what you mean; it appears to be some kind of incorrect transform. I would file a bug on the Solidworks glTF exporter. Maybe first double-check that other glTF readers show the same error: https://sandbox.babylonjs.com/
